Conceptpower+Spring/src/main/webapp/WEB-INF/views/layouts/concepts/conceptlistreferences.html

```diff
@@ -0,0 +1,115 @@
+<html
+   xmlns:th="http://www.thymeleaf.org"
+   xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" 
+   layout:decorate="~{layouts/main}"
+   >
+   <head>
+      <script>
+         $(document).on('click', '#conceptDetails', function(e) {
+             var conceptid = $(this).data('conceptid');
+         
+             $.ajax({
+                 type: "GET",
+                 url: "/conceptpower/conceptDetail",
+                 data: {
+                     conceptid: conceptid
+                 },
+                 success: function(details) {
+                     details = $.parseJSON(details);
+                     var token = $("meta[name='_csrf_token']").attr("content");
+                 	
+                     //Ajax Call to plugin a pre populated Concepet Details Modal
+                     $.ajax({
+                         type: "POST",
+                         url: "/conceptpower/conceptdetails?_csrf=" + token.trim(),
+                         data: JSON.stringify(details),
+                         contentType: "application/json",
+                         success: function(data) {
+                             $('#conceptDetailsPlugin').html(data);
+                             $('#conceptDetail').modal('show');
+                         }
```

Comment:
what is this ajax call doing? The controller is just returning the details, isn't it? It's pretty unsafe since you use `html` to show the content, which would probably execute any javascript in `data`. If someone  manages to insert malicious javascript into a concept, that could have bad consequences. Why don't you just show the details, since you already have them?

Reply:
Sure. I will do that.